epoch.go

```diff
@@ -0,0 +1,23 @@
+package kapeta
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Epoch is a custom type for time.Time that marshals and unmarshals to/from epoch time in milliseconds.
+type Epoch time.Time
+
+func (m Epoch) MarshalJSON() ([]byte, error) {
```

Comment:
I guess you might wanna be working on the pointer here as in m *Epoch?

Perhap a test to verify that it works :)